### ISSUE TITLE
[Feature Request] Auto Quarantine files if threat detected !

### ISSUE THREAD

**Issue by TecnikOfficial** [Divested-Mobile/hypatia#39]
i tested [hypatia-test-file.png](https://divestos.org/images/hypatia-test-file.png) but nothing happened and yes my realtime protection is on . Quarantine section or scan log section is very much needed

**Comment by SkewedZeppelin** [comment]
You save the file and nothing happens? What browser did you use?

Is the "realtime scanner" notification shown?

**Comment by TecnikOfficial** [comment]
yes notification is shown and i use [adblock browser](https://play.google.com/store/apps/details?id=org.adblockplus.browser) by abp

**Comment by ghost** [comment]
I use Fennec and Brave, the issue is the same on both.

After the file is saved, hypatia notification appears but, since there is no auto quarantine, you can open the infected file from:

- the browser notification
- any file manager
- for images, any gallery app

Without any real block, the user is exposed to a possible infection

### open file from browser notification, at the bottom
![Imagepipe_0](https://github.com/Divested-Mobile/Hypatia/assets/15078341/115ffbc5-4931-4cad-9cf3-3b00dceead79)

**Comment by AdityaMH** [comment]
Also you can't delete it from notifications

![Screenshot_20240818-050209](https://github.com/user-attachments/assets/c7b3decc-5320-4a3c-b941-bd6f4c566949)

For this file

**Comment by RedMentos** [comment]
Auto blocking or quarantine files i wish too.
Thats a huge thing to miss.
